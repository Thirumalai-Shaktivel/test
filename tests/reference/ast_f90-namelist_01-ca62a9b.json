{
    "basename": "ast_f90-namelist_01-ca62a9b",
    "cmd": "lfortran --show-ast-f90 --no-color {infile}",
    "infile": "tests/../integration_tests/namelist_01.f90",
    "infile_hash": "82b81ebf89597c4cee2bae25cbe602a58d7d628cf9dec14337630861",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast_f90-namelist_01-ca62a9b.stdout",
    "stdout_hash": "ad122de5449b166569b8ac2291aa5e560d6401b69a6a8f4877c8176e",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}