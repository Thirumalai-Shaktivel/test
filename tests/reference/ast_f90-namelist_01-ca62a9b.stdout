program namelist1
integer :: N
real :: t
real :: x
real :: Ecut
namelist /domain/ N, t, x, Ecut
end program namelist1
